<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CABBH8E" failures="1" tests="5" name="Home 2" time="0.123" errors="0" timestamp="2025-06-06T16:54:08 TRT">
  <testcase classname="Test_Package.Basic_TestNG_2" name="SecondDemo" time="0.029">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test_Package.Basic_TestNG_2.SecondDemo(Basic_TestNG_2.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- SecondDemo -->
  <testcase classname="Test_Package.Basic_TestNG_2" name="ThirdDemo" time="0.004"/>
  <testcase classname="Test_Package.Basic_TestNG_4" name="MobileAppLogin" time="0.002"/>
  <testcase classname="Test_Package.Basic_TestNG_4" name="WebLogin" time="0.003"/>
  <testcase classname="Test_Package.Basic_TestNG_4" name="LoginAPI" time="0.003"/>
</testsuite> <!-- Home 2 -->
